qt/Prefs.cc

```diff
@@ -148,6 +148,9 @@ std::array<Prefs::PrefItem, Prefs::PREFS_COUNT> const Prefs::Items{
     { RPC_ENABLED, TR_KEY_rpc_enabled, QMetaType::Bool },
     { RPC_PASSWORD, TR_KEY_rpc_password, QMetaType::QString },
     { RPC_PORT, TR_KEY_rpc_port, QMetaType::Int },
+    { RPC_SSL_CERT, TR_KEY_rpc_ssl_cert, QMetaType::QString },
```

Comment:
I don't see you using these preferences in the Qt client, so I wonder if you even want/need to add them here.

Reply:
ok I removed this part